content/en/docs/components/pipelines/user-guides/core-functions/cli.md

```diff
@@ -1,6 +1,7 @@
 +++
-title = "Interact with KFP via the CLI"
-weight = 4
+title = "Use the KFP CLI"
+description = "Learn how to interact with Kubeflow Pipelines using the KFP CLI."
+weight = 203
```

Comment:
Just curious: Why did you set this weight

Reply:
Because it should go down the bottom of the section, it's effectively a "reference" doc.

We can discuss moving it to the actual reference section in a future PR, but I don't want to move any pages in this PR.